pkgs/tools/misc/heatseeker/default.nix

```diff
@@ -13,6 +13,13 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "0jnlcm7v29m4nc318qgf7r7jvs80s7n04fw83imm506vwr9rxbx9";
 
+  # https://github.com/rschmitt/heatseeker/issues/42
+  # I've suggested using `/usr/bin/env stty`, but doing that isn't quite as simple
+  # as a substitution, and this works since we have the path to coreutils stty.
+  patchPhase = ''
+    substituteInPlace src/screen/unix.rs --replace "/bin/stty" "${coreutils}/bin/stty" 
```

Comment:
You need to add `coreutils` as an argument (on the first line) to make `ofborg` happy again.